sparkprs/controllers/tasks.py

```diff
@@ -51,6 +51,21 @@ def backfill_prs():
     return "Enqueued tasks to backfill %i PRs" % latest_pr_number
 
 
+@tasks.route("/github/clean-prs")
+def clean_deleted_prs():
+    prs = Issue.query(Issue.state == "open")
+    pr_list = ""
+    for pr in prs:
+        try:
+            raw_github_request(get_pulls_base() + '/%i' % pr.number,
+                               oauth_token=oauth_token, etag=pr.etag)
+        except:
```

Comment:
It might be nice to put this logic into `Issue.refresh()` function. Once we do that, the `clean_deleted_prs` function can just be changed to be something which enqueues update tasks for all open PRs to ensure that we try to update any PR at least once per day.